automation-swiftui/AutomationAssessment/ContentView.swift

```diff
@@ -30,20 +23,21 @@ struct ContentView: View {
                 }
             } label: {
                 Image(systemName: "arrow.clockwise.circle.fill")
+                    .imageScale(.large)
```

Comment:
This is for accessibility- I think the refresh was too small a touch target, and might be hard to see for visually impaired candidates